docs/source/_ecf_usage_participant_actions.html.erb

```diff
@@ -65,7 +65,7 @@
   <tbody>
     <tr class="govuk-table__row">
       <td class="govuk-table__cell">Output 1 - Participant Start (20%)</td>
-      <td class="govuk-table__cell">31st October 2021</td>
```

Comment:
@asmega Can you check this with Conor because I don't think it makes sense in this context? It suggests we'd accept until the 30th of November and then pay the same day.

Reply:
I will confirm, probably will change to 17th/19th or something in that region

Reply:
i've now set this to the 19th